apps/bundle_deploy/build_model.py

```diff
@@ -33,19 +34,21 @@ def build_module(opts):
     func = mod["main"]
     func = relay.Function(func.params, relay.nn.softmax(func.body), None, func.type_params, func.attrs)
 
-    with tvm.transform.PassContext(opt_level=3):
+    with tvm.transform.PassContext(opt_level=3, config={'tir.disable_vectorize': True}):
         graph, lib, params = relay.build(
-            func, 'llvm --system-lib', params=params)
+            func, 'c', params=params)
```

Comment:
why should we change this default behavior?

Reply:
llvm didn't work on my mac, but I can leave this alone at HEAD if it works for others.

--system-lib is being handled differently now (I.e. by `python/tvm/micro/function_registry.py`) but should be able to fold that in to C++ (I.e. as something like  `--system-lib --runtime=crt`) in a follow-on.

Reply:
I suggest keeping `llvm --system-lib`, since we are targeting uTVM, not mac :-)